.storybook/blocks/PropertiesTable.jsx

```diff
@@ -0,0 +1,65 @@
+import "@spectrum-css/table";
+
+import { DocsContext, useOf } from "@storybook/blocks";
+import { ResetWrapper } from "@storybook/components";
+import { NAVIGATE_URL } from "@storybook/core-events";
+import { styled } from "@storybook/theming";
+import React, { useContext } from 'react';
+import { Body, Code, LinkableHeading } from "./Typography.jsx";
+
+export const Table = styled.table`
+	padding-block: 10px;
+`;
+
+/**
+ * Displays the modifiable custom properties for a component based on the metadata provided in the story.
+ * The story metadata object is imported from the "metadata.json" file in the component's directory.
+ *
+ * If the metadata object does not contain a "modifiers" array, this component will not render.
```

Comment:
Thinking back to the field group (which doesn't have any modifiers in the CSS), should we remove the `"modifiers": []` from field group metadata.json? Right now, I see an empty table, but it sounds like I shouldn't.
<img width="1065" alt="Screenshot 2024-10-09 at 11 03 27 AM" src="https://github.com/user-attachments/assets/92678a30-3ac9-4a5a-b2d6-0983db27c479">

OR if this we had `!metadata?.modifiers.length` [here](https://github.com/adobe/spectrum-css/blob/8cfec9a60a8bdbe7c6fe86a6bba59796e16fecef/.storybook/blocks/PropertiesTable.jsx#L31), it wouldn't return the mods table.

Reply:
Good idea!  I added that and pushed up the update.